src/shared/lib/geography/country-levels.js

```diff
@@ -74,6 +75,10 @@ export const getFeature = async id => {
       },
       geometry: newGeometry
     };
+    if (newGeometry.coordinates.length === 0) {
+      console.error(`Combined geometry for ${newFeature.properties.name} (${id}) does not contain any coordinates!`);
```

Comment:
Can you make this an assert?
We should not pass on an empty geometry, it's a bug in country-levels if one is empty.

Reply:
Addressed.